CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 1.0.3
```

Comment:
I know we didn't do this for the corresponding input plugin, but I think we probably should increase at least the minor version given that we have removed compatibility with a large number of Logstash versions